TicTacToe.cpp

```diff
@@ -20,3 +21,33 @@ TicTacToe::TicTacToe()
     
     victory = 0;    //set victory to false
  }
+
+bool TicTacToe::add(int x, int y)
+{
+    // if there is at least one ball to remove, remove ball from bottom of the pole
+    if (field[y][x].size() > 0) {
```

Comment:
I think you switched the conditions around here. This should be the check in `remove()4